atlasdb-workload-server-antithesis/tag-test-images.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo "Preparing images for Antithesis test"
+
+VERSION=$1
+EXPECTED_ANTITHESIS_TEST_TAG="unspecified"
+
+docker pull palantirtechnologies/cassandra:2.2.18-1.116.0
+docker tag palantirtechnologies/cassandra:2.2.18-1.116.0 palantirtechnologies/cassandra:${EXPECTED_ANTITHESIS_TEST_TAG}
```

Comment:
yeah it's a bit unfortunate that this is duplicated in the actual Antithesis publish script, though a nice solution seems like a fair bit more work